rest/dao/src/main/resources/io/syndesis/dao/deployment.json

```diff
@@ -755,31 +756,100 @@
           "componentProperty": true,
           "defaultValue": "sampledb",
           "description": "Database schema"
-        },
-        "catalog": {
-          "kind": "property",
-          "displayName": "Catalog",
-          "group": "common",
-          "required": false,
-          "type": "string",
-          "javaType": "java.lang.String",
-          "tags":[],
-          "deprecated": false,
-          "secret": false,
-          "componentProperty": true,
-          "description": "Database catalog"
         }
       },
       "actions": [
         {
-          "id": "io.syndesis:sql-stored-connector:latest",
-          "name": "Invoke SQL stored procedure",
-          "description": "Invoke SQL stored procedure to get data in and/or out",
+          "id": "sql-connector",
+          "name": "Invoke SQL",
+          "description": "Invoke SQL to get data in and/or out",
+          "tags": [ "dynamic" ],
+          "pattern": "To",
+          "actionType": "connector",
+          "descriptor": {
+            "camelConnectorGAV": "io.syndesis:sql-connector:1.2-SNAPSHOT",
+            "camelConnectorPrefix": "sql-connector",
+            "inputDataShape": {
+               "kind" : "json-schema"
+            },
+            "outputDataShape": {
+               "kind" : "json-schema"
+            },
+            "propertyDefinitionSteps": [{
+              "name": "SQL Statement",
+              "description": "Enter a SQL Statement starting with INSERT, SELECT, UPDATE or DELETE",
+              "properties": {
+                  "query":{
+                    "kind":"path",
+                    "displayName":"Query",
+                    "group":"common",
+                    "required":true,
+                    "type":"string",
+                    "javaType":"java.lang.String",
+                    "deprecated":false,
+                    "secret":false,
+                   "description":"SQL Statement to be executed. Can contain input parameters prefixed by ':#' i.e. ':#MYPARAMNAME'"
+                 }
+               }
+            }]
+          }
+        },
+        {
+          "id": "sql-start-connector",
+          "name": "Periodic SQL Invocation",
+          "description": "Invoke SQL to periodically get data out",
+          "tags": [ "dynamic" ],
+          "pattern": "From",
+          "actionType": "connector",
+          "descriptor": {
+            "camelConnectorGAV": "io.syndesis:sql-start-connector:1.2-SNAPSHOT",
```

Comment:
How to deal with depending on a SNAPSHOTS? How does this work in the mono repo? Hardcoding for now..